src/Components/ActivationKeys/activation-keys-table.js

```diff
@@ -0,0 +1,163 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  sortable,
+} from '@patternfly/react-table';
+import {
+  Button,
+  Flex,
+  FlexItem,
+  Grid,
+  GridItem,
+  Text,
+  TextContent,
+  Title,
+} from '@patternfly/react-core';
+import { CopyIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
+import SkeletonTable from '@redhat-cloud-services/frontend-components/SkeletonTable';
+import CopyCommand from './copy-command';
+
+const mockGetData = () =>
+  new Promise((res) =>
+    setTimeout(
+      () =>
+        res([
+          {
+            name: 'sample-name-1',
+            role: 'Red Hat Linux Server',
+            level: 'Premium',
+            usage: 'Production',
+          },
+          {
+            name: 'sample-name-2',
+            role: 'Red Hat Linux Workstation',
+            level: 'Standard',
+            usage: 'Development/Test',
+          },
+          {
+            name: 'sample-name-3',
+            role: 'Red Hat Linux Compute Node',
+            level: 'Self-Suport',
+            usage: 'Disaster Recovery',
+          },
+        ]),
+      250
+    )
+  );
+
+const columns = [
+  { title: 'Name', transforms: [sortable] },
+  { title: 'Role', transforms: [sortable] },
+  { title: 'Service level', transforms: [sortable] },
+  { title: 'Usage', transforms: [sortable] },
+  { title: '' },
+];
+
+function ActivationKeysTable() {
+  const [data, setData] = useState(undefined);
+  const [orgId, setOrgId] = useState(undefined);
+  const [sortBy, setSortBy] = useState({ index: 0, direction: 'asc' });
+
+  const handleTableButtonClick = (name, data) => {
+    const row = data.find((row) => row.name === name);
+    const elem = document.createElement('input');
+    elem.id = `temp-${name}`;
+    elem.type = 'text';
+    elem.value = JSON.stringify(row);
+    document.body.appendChild(elem);
+    document.getElementById(`temp-${name}`).select();
+    document.execCommand('copy');
+    document.body.removeChild(elem);
```

Comment:
maybe we could make this seperate function so it will be reusable

Reply:
Maybe. I honestly don't understand why this is not available as a standalone component in PF. Only tied to that magnificent input.

Reply:
They export the function in the file.. so maybe it's reusable. 🤔